app/shared/resource-map/mapSelector.spec.js

```diff
@@ -57,8 +57,7 @@ describe('shared/resource-map/mapSelector', () => {
       const props = { resourceIds: ['123', '321'] };
       const data = selector(state, props);
       expect(data.markers).to.deep.equal([
-        { latitude: 0, longitude: 1, resourceId: '123' },
-        { latitude: 0, longitude: 1, resourceId: '321' },
+        { unitId: 21, latitude: 0, longitude: 1, resourceIds: ['123', '321'] },
```

Comment:
- [ ] Adding one more resource with a different unit to this test would be nice.